pkg/minikube/download/preload.go

```diff
@@ -141,6 +142,20 @@ func Preload(k8sVersion, containerRuntime string) error {
 	out.Step(style.FileDownload, "Downloading Kubernetes {{.version}} preload ...", out.V{"version": k8sVersion})
 	url := remoteTarballURL(k8sVersion, containerRuntime)
 
+	checksum, err := getChecksum(k8sVersion, containerRuntime)
+	var realPath string
+	if err != nil {
+		klog.Warningf("No checksum for preloaded tarball for k8s version %s", k8sVersion)
```

Comment:
please log the error there, in addition to the k8sVersion

Reply:
Done